cypress/support/pages/yves/login/repositories/suite-login-repository.ts

```diff
@@ -15,7 +15,8 @@ export class SuiteLoginRepository implements LoginRepository {
   getRegisterConfirmPasswordInput = (): Cypress.Chainable => cy.get('#registerForm_password_confirm');
   getRegisterAcceptTermsCheckbox = (): Cypress.Chainable => cy.get('#registerForm_accept_terms');
   getRegisterForm = (): Cypress.Chainable => cy.get('form[name=registerForm]');
-  getFailedAuthenticationText = (): string => 'Authentication failed';
+  getFailedAuthenticationText = (): string =>
+    'Please check that your E-mail address and password are correct and that you have confirmed your E-mail address by clicking the link in the registration message';
```

Comment:
what about `b2c-mp` and `b2b-mp` repos?

Reply:
b2b doesn't have customer login I believe. For b2b-mp will add

Reply:
I believe that customer login functionality was implemented everywhere.

Reply:
if b2bmp have customer login, then b2b have for sure

Reply:
Ok, updated for both